src/node/plugin.ts

```diff
@@ -384,6 +384,7 @@ export async function createVitePressPlugin(
         try {
           await resolveUserConfig(siteConfig.root, 'serve', 'development')
         } catch (err: any) {
+          console.error(err)
```

Comment:
can you refactor it to use `siteConfig.logger.error` instead

Reply:
Ok.